cli/src/program.rs

```diff
@@ -2364,13 +2429,23 @@ fn do_process_program_upgrade(
     buffer_pubkey: &Pubkey,
     buffer_signer: Option<&dyn Signer>,
     skip_fee_check: bool,
+    max_retries: Option<usize>,
+    tx_priority_fee: Option<u64>,
 ) -> ProcessResult {
     let blockhash = rpc_client.get_latest_blockhash()?;
 
     let (initial_message, write_messages, balance_needed) =
         if let Some(buffer_signer) = buffer_signer {
+            let mut initial_instructions: Vec<Instruction> = Vec::new();
+
+            // Set compute unit price
+            if let Some(tx_priority_fee) = tx_priority_fee {
+                initial_instructions.push(ComputeBudgetInstruction::set_compute_unit_price(
```

Comment:
nit: perhaps rename the new arg to `compute-unit-price` (instead of `tx_priority_fee`) to be consistent with `CompuyeBudgetInstruction` it invokes.

Reply:
yup will do

Reply:
resolved